Assets/Plugins/Source/Editor/ConfigEditors/ConfigEditor.cs

```diff
@@ -38,26 +39,101 @@ namespace PlayEveryWare.EpicOnlineServices.Editor
     /// <typeparam name="T">Intended to be a type accepted by the templated class EOSConfigFile.</typeparam>
     public class ConfigEditor<T> : IConfigEditor where T : EpicOnlineServices.Config
     {
+        /// <summary>
+        /// The string to use for the label for the config editor.
+        /// </summary>
         private readonly string _labelText;
+
+        /// <summary>
+        /// Action that is called when the config editor is expanded.
+        /// </summary>
+        public Action<ConfigEditor<T>> OnExpanded;
+
+        /// <summary>
+        /// Used to animate the expansion and collapse of the config editor if
+        /// doing so is enabled.
+        /// </summary>
+        public AnimBool _animExpanded;
```

Comment:
As these are public, should they be named something else? Or is this particular name important?

Reply:
Good catch. At some point while developing, I had this temporarily be public so I could more easily play with it. I neglected to subsequently return it to `private`. I've done that now, so marking this conversation as resolved.